src/layers/temporalconv.jl

```diff
@@ -186,3 +186,42 @@ end
 function Base.show(io::IO, a3tgcn::A3TGCN)
     print(io, "A3TGCN($(a3tgcn.in) => $(a3tgcn.out))")
 end
+
+@doc raw"""
+    TemporalGraphConv(layer)
+
+A constructor for applying to each snapshot of the `TemporalSnapshotsGNNGraph` a convolutional homogeneous graph `layer`.
+
```

Comment:
Paper citation is missing, and also more explanation of how it works. 

Am I correct in saying that this applies `layer` independently to each temporal slice?

Reply:
>  I would say it is similar to the HeteroGraphConv for heterographs.

Yes, although HeteroGraphConv in the end aggregates messages coming from different edge types, while here there is no cross-talk between different times, it has to happen in separate modules (e.g. temporal attention in the last reference above)